src/3rdParty/lazy_loader/CMakeLists.txt

```diff
@@ -8,4 +9,5 @@ SET_PYTHON_PREFIX_SUFFIX(lazy_loader)
 
 fc_copy_sources(lazy_loader "${CMAKE_BINARY_DIR}/Ext/lazy_loader" ${lazy_loader})
 install (FILES lazy_loader.py DESTINATION "${CMAKE_INSTALL_PREFIX}/Ext/lazy_loader")
+install (FILES __init__.py DESTINATION "${CMAKE_INSTALL_PREFIX}/Ext/lazy_loader")
```

Comment:
You don't need to add every file to the install instruction. Since you defined a variable, you can do this.
```
INSTALL(FILES ${lazy_loader} DESTINATION ...)
```